src/Database/Adapter/MariaDB.php

```diff
@@ -1632,6 +1632,79 @@ public function deleteDocument(string $collection, string $id): bool
         return $deleted;
     }
 
+    /**
+     * Delete Documents
+     *
+     * @param string $collection
+     * @param array<string> $ids
+     *
+     * @return bool
+     */
+    public function deleteDocuments(string $collection, array $ids): bool
+    {
+        if (empty($ids)) {
+            return true;
+        }
+
+        try {
+            $name = $this->filter($collection);
+            $where = [];
+
+            if ($this->sharedTables) {
+                $where[] = "_tenant = :_tenant";
+            }
+
+            $where[] = "_uid IN (" . \implode(', ', \array_map(fn ($id) => ":_id_{$id}", \array_keys($ids))) . ")";
```

Comment:
Let's use index here 

```suggestion
            $where[] = "_uid IN (" . \implode(', ', \array_map(fn ($index) => ":_id_{$index}", \array_keys($ids))) . ")";
```